homeassistant/components/plex/media_player.py

```diff
@@ -407,7 +407,21 @@ def refresh(self, device, session):
                 self._make = self._player.device
             else:
                 self._is_player_available = False
-            self._media_position = self._session.viewOffset
+
+            # Calculate throttled position for proper progress display.
+            position = int(self._session.viewOffset / 1000)
+            now = dt_util.utcnow()
+            if self._media_position and self._media_position_updated_at:
```

Comment:
This seems to cause an extra update if `_media_position` was previously zero (i.e. the start of the media).

Since these two attributes always change together, maybe this suggestion is sufficient?

```suggestion
            if self._media_position is not None:
```

Reply:
This works well in testing, thanks!